### ISSUE TITLE
TRAVIS-CI: random tests are failing when they pass locally

### ISSUE THREAD

**Issue by RudyOnRails** [aic-collections/aicdams-lakeshore#465]
Take a look at:
* rspec seeding feature (vs defined)
* Lakeshore.restore method
* https://github.com/samvera/hyrax/wiki/Hyrax-Development-Guide#ive-pushed-my-branch-which-passed-locally-to-github-and-the-build-failed-on-travis-ci-what-gives-i-need-to-be-able-to-reproduce-this-locally-to-get-my-branch-merged

**Comment by RudyOnRails** [comment]
Ok, so this [build #2916](https://travis-ci.org/aic-collections/aicdams-lakeshore/builds/362354962)  runs green again after restarting. That is a sign that this build is consistent. I am going to look into the [diff/comparison](https://github.com/aic-collections/aicdams-lakeshore/compare/bfbd7ad52a9d...89c0a873eb1a) for that build's [HEAD](https://github.com/aic-collections/aicdams-lakeshore/commit/bfbd7ad52a9dd0ceed97c4b5af512c0887978dcc) and the next build's [HEAD](https://github.com/aic-collections/aicdams-lakeshore/commit/89c0a873eb1a406e62303e5ae7bcf2605d779315).

While I'm doing this, I'm going to once again restart the green [build](https://travis-ci.org/aic-collections/aicdams-lakeshore/builds/362354962) just for sanity since sometimes false negatives do go green. If we get two greens consecutively, then I think we're good. Maybe I'll do 3-4 😄 

Based on that [diff/comparison](https://github.com/aic-collections/aicdams-lakeshore/compare/bfbd7ad52a9d...89c0a873eb1a), I think I am going to switch:
```
CurationConcerns.config.callback.run(:after_update_content, file_set, user)
```
to the other option I was considering...
```
ContentNewVersionEventJob.perform_later(file_set, user)
```
and build that branch.

**Comment by RudyOnRails** [comment]
closing via https://github.com/aic-collections/aicdams-lakeshore/pull/466

**Comment by RudyOnRails** [comment]
reopening so i can close automatically once the PR with the keyword and issue number is merged into the default branch :)
